plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2010,13 +2039,17 @@ public void registerCoreMembers() {
         // @Group core
         //
         // @Description
-        // TODO: Document Command Details
+        // Injects a script into the current ScriptQueue.
+        // This means this task will run with all of the queues associated definitions and tags.
+        // It will also now be part of the queue, so any delays or definitions used in the injected script will be
+        // accessible in the original queue.
         //
         // @Tags
         // None
         //
         // @Usage
-        // TODO: Document Command Details
+        // - inject InjectedTask
+        // Injects the InjectedTask task into the current queue

```

Comment:
These two lines are out of order - the Usage block is ALWAYS one line "Use to do stuff.." then remaining lines are script code